prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,1155 @@
+{
+  "Framework": "AWS-Account-Security-Onboarding",
+  "Version": "",
+  "Provider": "AWS",
+  "Description": "Checklist when onboarding new AWS Accounts to existing AWS Organization.",
+  "Requirements": [
+    {
+      "Id": "Predefine IAM Roles",
+      "Description": "Check if exists predefine IAM Roles",
+      "Attributes": [
+        {
+          "Section": "Deploy account from predefined IaC template",
+          "Service": "IAM"
+        }
+      ],
+      "Checks": [
+        "iam_no_custom_policy_permissive_role_assumption",
+        "iam_policy_attached_only_to_group_or_roles",
+        "iam_support_role_created",
+        "iam_role_cross_service_confused_deputy_prevention"
+      ]
+    },
+    {
+      "Id": "Enabled security services",
+      "Description": "Check if security services are enabled",
+      "Attributes": [
+        {
+          "Section": "Deploy account from predefined IaC template",
+          "Service": "Security",
+          "Type": "Manual"
+        }
+      ],
+      "Checks": []
```

Comment:
There are checks that verify if there are security services enabled

Reply:
Done!